### DISCUSSION TITLE
Real-time collaboration ⚡️

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#210] [Ideas]
Right now we don't have any websocket in place to sync data in real-time but this is definitely something we will need in the future.

Example features we could build:
- Refresh people/companies/objects lists as they are updated
- Show with a color border when someone is editing a field on a record
- Update comments threads panel as comments are sent
- Update inbox page as inbox notifications are received (later / inbox page isn't built yet as I'm writing this)
- View who is viewing the same page (later / record show page isn't built yet as I'm writing this)

**Comment by wmbutler** [comment]
#18106 likely related. I was hoping websockets were already in place, but it appears not. This would be particularly useful for workflows that alter data on the backend so they could deliver the results to the frontend for display.

**Reply by FelixMalfait** [reply]
They are but you need to enable it in Settings > Updates. It is still in beta but will soon move to GA in the next release

**Comment by BOHEUS** [comment]
Closing as real-time updates are publicly available now https://github.com/twentyhq/twenty/pull/18114

**Reply by FelixMalfait** [reply]
The only thing we need to improve imo is Rich Text field collaboration ; we need special optimization/work for that part
